lib/Sema/TypeCheckDecl.cpp

```diff
@@ -4204,7 +4122,7 @@ class DeclChecker : public DeclVisitor<DeclChecker> {
             TC.checkTypeModifyingDeclAttributes(var);
 
           // Decide whether we should suppress default initialization.
-          if (isNeverDefaultInitializable(PBD->getPattern(i)))
+          if (!PBD->getPattern(i)->isDefaultInitializable())
```

Comment:
This is the one remaining call to `isNeverDefaultInitializable` on a `Pattern` outside of `PatternBindingDecl.cpp`